<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>718</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>718</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Request</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>165</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Led类容</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtLed"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btnRequest">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>请求</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>136</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>操作</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxOp">
        <item>
         <property name="text">
          <string>查询A地感状态</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>查询B地感状态</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>查询红外状态</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>发布LED</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>红绿灯控制</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>道闸控制</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>查询车牌数据</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>主动发送</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>94</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>红绿灯</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxTraffic">
        <item>
         <property name="text">
          <string>红灯</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>绿灯</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>关灯</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>165</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtAddress"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>82</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>道闸</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxGate">
        <item>
         <property name="text">
          <string>开闸</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>关闸</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btnContinue">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>连续开关闸</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Response</string>
    </property>
    <widget class="QTextEdit" name="edtResponse">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>251</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblPic">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>车牌服务器</string>
    </property>
    <widget class="QPushButton" name="btnConnect">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>链接车牌服务器</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>153</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>IP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtIP"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>165</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtPort"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
